e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -363,4 +363,24 @@ describeWithSnowplow("scenarios > dashboard > tabs", () => {
     saveDashboard();
     expectGoodSnowplowEvents(PAGE_VIEW_EVENT + 2); // dashboard_tab_deleted
   });
+
+  it(
+    "should send snowplow events when cards are moved between tabs",
+    { scrollBehavior: false },
```

Comment:
With the automatic scroll the "move tab" icon scrolls under the header, making the test fail.
Is there a better solution?

Reply:
(I'm currently checking if it's true that it's flaky, as I couldn't find the failed test on deploy sentinel to link here)